services/src/floodlight-modules/src/main/java/org/openkilda/floodlight/switchmanager/SwitchManager.java

```diff
@@ -1474,7 +1484,94 @@ public void sendPortUpEvents(final IOFSwitch sw) throws SwitchOperationException
                 }
             }
         }
+    }
+    
+    /**
+     * {@inheritDoc}
+     */
+    @Override
+    public PortConfigurationResponse configurePort(final PortConfigurationRequest request) 
+            throws SwitchOperationException {
+        DatapathId dpId = DatapathId.of(request.getSwitchId());
+        IOFSwitch sw = lookupSwitch(dpId);
+        
+        OFPortDesc ofPortDesc = getPort(sw, request.getPortNo());
+        logger.debug("Switch: {}, port: {}", sw, ofPortDesc);
```

Comment:
The message is absolutely not informative.

Reply:
Logging was not required. Removed.